source/install-from-source.txt

```diff
@@ -0,0 +1,373 @@
+
+.. important::
+
+  It is **highly recommended** that new users use a stable released version of
+  the driver, rather than building from a development branch. When you
+  ``git clone`` or download an archive of the repository, be sure to specify a
+  release tag (e.g. with Git's ``--branch`` argument).
+
+.. _learn.obtaining.git:
+
+Downloading Using Git
+~~~~~~~~~~~~~~~~~~~~~
+
+Using Git, the C driver repository can be cloned from the GitHub URL
+https://github.com/mongodb/mongo-c-driver.git. Git tags for released versions
+are named after the version for which they correspond (e.g. "{+full-version+}"). To
+clone the repository using the command line, the following command may be used:
+
+.. code-block:: bash
+
+  $ git clone https://github.com/mongodb/mongo-c-driver.git --branch="$VERSION" "$SOURCE"
+
+.. tip::
+
+  Despite the name, ``git-clone``'s ``--branch`` argument may also be used to
+  clone from repository *tags*.
+
+.. _learn.obtaining.archive:
+
+Downloading a Release Archive
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+An archived snapshot of the repository can be obtained from the
+`GitHub Releases Page <https://github.com/mongodb/mongo-c-driver/releases>`__.
+The ``mongo-c-driver-x.y.z.tar.gz`` archive attached to any release contains the
+minimal set of files that you'll need for the build.
+
+.. tabs::
+
+   .. tab:: Using ``wget`` + ``tar``
+      :tabid: wget-tar
+
+      .. code-block:: none
+        
+        ## Download using wget:
+        $ wget "https://github.com/mongodb/mongo-c-driver/archive/refs/tags/$VERSION.tar.gz" \
+             --output-document="mongo-c-driver-$VERSION.tar.gz"
+        ## Extract using tar:
+        $ tar xf "mongo-c-driver-$VERSION.tar.gz"
+
+   .. tab:: Using ``curl`` + ``tar``
+      :tabid: curl-tar
+
+      .. code-block:: none
+        
+        ## Using curl:
+        $ curl "https://github.com/mongodb/mongo-c-driver/archive/refs/tags/$VERSION.tar.gz" \
+             --output="mongo-c-driver-$VERSION.tar.gz"
+        ## Extract using tar:
+        $ tar xf "mongo-c-driver-$VERSION.tar.gz"
+
+   .. tab:: PowerShell
+      :tabid: powershell
+
+      .. code-block:: none
+
+        ## Use Invoke-WebRequest:
+        PS> $url = "https://github.com/mongodb/mongo-c-driver/archive/refs/tags/$VERSION.zip"
+        PS> $file = "mongo-c-driver-$VERSION.zip"
+        PS> Invoke-WebRequest -UseBasicParsing -Uri $url -OutFile $file
+        ## Extract using Expand-Archive:
+        PS> Expand-Archive mongo-c-driver-$VERSION.zip
+
+The above commands will create a new directory ``mongo-c-driver-$VERSION`` within
+the directory in which you ran the ``tar`` / ``Expand-Archive`` command (**note**:
+PowerShell will create an additional intermediate subdirectory of the same
+name). This directory is the root of the driver source tree (which we refer to
+as ``$SOURCE`` in these documents). The ``$SOURCE`` directory should contain the
+top-level ``CMakeLists.txt`` file.
```

Comment:
S: Since there are a couple of decision paths to represent here (tar / expand-archive / powershell), I would probably put a different block of text beneath each tab. (You can also use `include` statements to avoid repetition.) Maybe something like this for PowerShell, with some style guide edits included:

```suggestion
The preceding commands create the ``[intermediate directory name]`` directory 
 within the directory in which you ran them. The ``[intermediate directory]`` directory
 contains the ``mongo-c-driver-$VERSION`` directory, which is the root of the driver
 source tree. This documentation refers to this directory as ``$SOURCE``. 
 The ``$SOURCE`` directory contains the top-level ``CMakeLists.txt`` file.
```